extensions/kind/src/extension.ts

```diff
@@ -23,11 +23,15 @@ import * as os from 'node:os';
 import { runCliCommand, detectKind } from './util';
 import { KindInstaller } from './kind-installer';
 import { window } from '@tmpwip/extension-api';
+import type { ImageInfoUI } from 'renderer/src/lib/image/ImageInfoUI';
```

Comment:
we can't import renderer stuff from an extension
extension should only rely on the public API